### PR TITLE
Fixed syntax to work with rustc 1.0.0-nightly

### PR DESCRIPTION
**Pull request by nawns**
rustc 1.0.0-nightly (b0aad7dd4 2015-03-22) (built 2015-03-22)

all 21 test cases pass

### CODE REVIEW

**Comment by gong023** [comment]
Awesome!
I think travis-ci doesn't pass because rustdoc is still wrong, but these changes are very useful.
Thanks a lot! :sparkles:

**Comment by gong023** [comment]
Travis passed successfully. Brilliant!

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "underscore"
-version = "0.0.2"
+version = "0.0.21"
```
Comment (benjycui) [inline Cargo.toml]: The version should be 0.0.3, right?

Comment (gong023) [inline Cargo.toml]: Ah, you are right. I had overlooked.
But I think there is no problem to work.
cf: I think version number should follow [Semantic Versioning](http://semver.org/).

**Pull request merged**